toolchain/newlib/crt0.S

```diff
@@ -18,6 +18,7 @@
 .global _start
 
 _start:
+    andq $-10, %rsp
```

Comment:
Can you use hex for this operand? It may make it clearer. Also please add a comment explaining why it's needed